packages/lesswrong/components/common/ForumIcon.tsx

```diff
@@ -239,6 +244,9 @@ const styles = (_: ThemeType): JssStyles => ({
   },
   linkRotation: {
     transform: "rotate(-45deg)",
+    '&.MuiListItemIcon-root': {
+      marginRight: 12
+    }
```

Comment:
This is to fix the spacing to the right of LW's link icon, so that the label text lines up with the other rows:
<img width="576" alt="Screen Shot 2023-06-16 at 9 30 32 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/481c8e32-0251-4300-af8c-a2cf6f17cec7">

Reply:
This seems mildy risky to me, but I'll trust that you've checked it doesn't break stuff elsewhere.

Reply:
This should only affect the LW link icon when using `ListItemIcon`, so I believe it shouldn't affect anywhere else on the site 👍